docs/contributing.md

```diff
@@ -131,7 +134,57 @@ These are listed roughly in order of decreasing importance.
    use `nextest` with `insta`,
    use `cargo insta test --workspace --test-runner nextest`.
 
- ## Modifying protobuffers (this is not common)
+## Previewing the HTML documentation
+
+<!---- Short-term TODO: More docs follow in a subsequent commit ---->
+
+### Setting up the prerequisites
+
+To build the website, you must have Python and `poetry` installed. If
+your distribution packages `poetry`, something like `apt install
+python3-poetry` is likely the best way to install it. Otherwise, you
+can download Python form <https://python.org> or follow the [Python
+installation instructions]. Finally, follow the [Poetry installation
+instructions].
+
+[Python installation instructions]: https://docs.python.org/3/using/index.html
+[Poetry installation instructions]: https://python-poetry.org/docs/#installation 
+
+Once you have `poetry` installed, you should ask it to install the rest
+of the required tools into a virtual environment as follows:
+
+    poetry install
```

Comment:
For consistency with other files, use a single backquoted block here instead of 4 spaces.